pkg/volume/util/util.go

```diff
@@ -144,11 +159,31 @@ func PathExists(path string) (bool, error) {
 		return true, nil
 	} else if os.IsNotExist(err) {
 		return false, nil
+	} else if isCorruptedMnt(err) {
+		return true, err
 	} else {
 		return false, err
 	}
 }
 
+// isCorruptedMnt return true if err is about corrupted mount point
+func isCorruptedMnt(err error) bool {
```

Comment:
this can be refactored
```go
func isCorruptedMnt(err error) bool {
  return err != nil && (err == syscall.ENOTCONN || err == syscall.ESTALE)
}

Reply:
`if err == nil {
  return false
}`
if We have check err ahead,it is not need to check it when return